local-requirements.txt

```diff
@@ -1,24 +1,24 @@
 auditwheel==5.1.2
-autobahn==22.3.2
-black==22.3.0
-flake8==4.0.1
+autobahn==22.7.1
+black==22.6.0
+flake8==5.0.4
 flaky==3.7.0
-mypy==0.942
+mypy==0.971
 objgraph==3.5.0
-Pillow==9.1.1
+Pillow==9.2.0
 pixelmatch==0.3.0
-pre-commit==2.18.1
+pre-commit==2.20.0
 pyOpenSSL==22.0.0
-pytest==7.1.1
-pytest-asyncio==0.18.3
+pytest==7.1.2
+pytest-asyncio==0.19.0
 pytest-cov==3.0.0
 pytest-repeat==0.9.1
 pytest-timeout==2.1.0
 pytest-xdist==2.5.0
-requests==2.27.1
+requests==2.28.1
 service_identity==21.1.0
-setuptools==60.9.3
-twine==3.8.0
+setuptools==63.4.2
+twine==4.0.1
```

Comment:
Should be fine, they just dropped Python 3.6 support see [here](https://twine.readthedocs.io/en/stable/changelog.html#twine-4-0-0-2022-03-31).